server/conf/routes

```diff
@@ -177,6 +177,8 @@ POST    /programs/:programId/blocks/:blockId/:inReview/:applicantRequestedAction
 # the path parameter as a string and decide how to handle it in the
 # controller.
 GET     /programs/:programParam     controllers.applicant.ApplicantProgramsController.show(request: Request, programParam: String)
+# Display error page if program is disabled (applicant access attempt).
+GET     /disabled          controllers.applicant.ApplicantProgramsController.showInfoDisabledProgram(request: Request)
```

Comment:
Added some more context hope it's clear :)

Reply:
Oh, sorry. I mean to say is that the url path itself should be more self-describing.

So like this `http://localhost/disabled` is ambiguous being right at the top level. I don't see this page being reused if a question or the application where disabled since they might need to do different things or at least have different copy.

compare to have a deeper and more descriptive path like this `http://localhost/programs/123/disabled` which indicates that program 123 is disabled.

Reply:
Ah gotcha! that makes sense. changes applied.